libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -240,9 +278,12 @@ float AP_BattMonitor::voltage2() const
 }
 
 /// current_amps - returns the instantaneous current draw in amperes
-float AP_BattMonitor::current_amps(uint8_t instance) const {
+float AP_BattMonitor::current_amp(uint8_t instance) const {
     if (instance < AP_BATT_MONITOR_MAX_INSTANCES) {
-        return _BattMonitor_STATE(instance).current_amps;
+        // Noticed that for some sensor types like the AttoPilot 90/180 the reported current can be negative under low load conditions
+        // might be because of a parasitic current, if the sensor is connected by a UBEC to the sensor board or whatever
+        // However, the current draw cannot be lower than zero, even if motors are breaking 
+        return fabs(_BattMonitor_STATE(instance).current_amps);
```

Comment:
I don't think fabs() is what you want here. Firstly, it should be fabsf(). Secondly, negative values should be zero instead of rectified to positive.